packages/governance/src/index.js

```diff
@@ -1,3 +1,9 @@
+// XXX ambient types runtime imports
+import '@agoric/ertp/exported.js';
+import '@agoric/zoe/src/contractFacet/types-ambient.js';
+import '@agoric/zoe/tools/types-ambient.js';
+import '@agoric/zoe/src/types.js';
```

Comment:
That's an unfortunate regression... but I see those imports are also in agoric-upgrade-13 ... well, a similar list.

fwiw, as of agoric-upgrade-13, the comment at the top cited #6512 ; might be nice to put that back